content/en/tracing/trace_collection/library_config/dotnet-framework.md

```diff
@@ -271,37 +309,15 @@ Enables or disables all instrumentation. Valid values are: `true` or `false`.<br
 : Sets status code ranges that will cause HTTP server spans to be marked as errors. <br>
 **Default**: `500-599`
 
+#### Logs
```

Comment:
BTW this Logs section is accurate, since the `DD_LOGS_INJECTION` configuration below corresponds to our integration with the Logs product